lib/schema/manifest.go

```diff
@@ -506,9 +513,19 @@ func (d Dependencies) ByName(names ...string) (*loc.Locator, error) {
 	return nil, trace.NotFound("dependencies %q are not defined in the manifest", names)
 }
 
+// Has returns true if dependencies have the specified package
```

Comment:
Godoc indicates bool return value, but function returns Locator.

Reply:
Good catch. Will remove since it is unused.